Grand Circus - OOP/GrandCircus/CircusModel/Circus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using iQuest.GrandCircus.Presentation;
+using System.Collections.Generic;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal class Circus
+    {
+        List<IAnimal> animals=new List<IAnimal>();
```

Comment:
Please make the animals field 'readonly'. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly